csharp/ql/src/Language Abuse/ForeachCapture.ql

```diff
@@ -5,9 +5,9 @@
  * @problem.severity recommendation
  * @precision medium
  * @id cs/captured-foreach-variable
- * @tags portability
- *       maintainability
- *       language-features
+ * @tags quality
+ *       reliability
+ *       correctness
```

Comment:
I'm not convinced we should add this query; the semantics of capturing a `foreach` variable has been stable since [C# 5](https://learn.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-version-history#c-version-50), which was released in 2012.